### ISSUE TITLE
Readme.md

### ISSUE THREAD

**Issue by nhrones** [AkiaCode/Packages#3]
Please add a readme.md file.
Perhaps a couple example URI ...
```
https://packages.deno.dev/search/:http?lang=deno
```
Thank you for this excellent example.

**Comment by AkiaCode** [comment]
@nhrones , 
```json
{
  "message": "Welcome to Packages API!",
  "endpoints": [
    {
      "name": "Hello",
      "description": "Returns a message",
      "path": "/"
    },
    {
      "name": "Information",
      "description": "Returns information about the package",
      "path": "/info/:packageName",
      "params": [
        {
          "lang": "Available langs: deno, nodejs, elixir, rust"
        }
      ]
    },
    {
      "name": "Search",
      "description": "Returns a list of packages",
      "path": "/search/:packageName",
      "params": [
        {
          "lang": "Available langs: deno, nodejs, elixir, rust"
        }
      ]
    }
  ]
}
```

**Comment by AkiaCode** [comment]
## Example
```
https://packages.deno.dev/search/harmony?lang=deno
https://packages.deno.dev/info/harmony?lang=deno
```

**Comment by AkiaCode** [comment]
Fixed: b34e255
